android/engine/src/main/java/org/smartregister/fhircore/engine/ui/appsetting/AppSettingActivity.kt

```diff
@@ -55,7 +55,9 @@ class AppSettingActivity : AppCompatActivity() {
             accountAuthenticator = accountAuthenticator
           ) { loadSuccessful: Boolean ->
             if (loadSuccessful) {
-              sharedPreferencesHelper.write(APP_ID_CONFIG, applicationId)
+              if (appSettingViewModel.rememberApp.value == true) {
```

Comment:
@pld Because of [these](https://github.com/opensrp/fhircore/issues/1070#issuecomment-1048491483) updates. The remember me functionality is only relevant when running the app in an online setting. Does it makes sense to update the code to online for these conditions or should we remove the remember me piece completely?

Reply:
Is remember me a user controlled setting? So this isn't needed because it's basically always active? If the app has configured pin login to be off, would it be useful then?

Reply:
Not really since we still need to support offline login capabilities